db/migrate/20170217085547_add_type_to_cloud_object_store_container.rb

```diff
@@ -0,0 +1,5 @@
+class AddTypeToCloudObjectStoreContainer < ActiveRecord::Migration[5.0]
+  def change
+    add_column :cloud_object_store_containers, :type, :string
```

Comment:
Thanks for adding this @miha-plesko 

Could you change this to just a single index used by both:

```ruby
add_index :cloud_object_store_containers, [:type, :ems_id]
```